### PR TITLE
Improve blog cards styling

### PR DESCRIPTION
**Pull request by clari182**
[Deploy preview](https://deploy-preview-1039--staging-aiid.netlify.app/)

Uses tailwind styled based cards for the blog with a new "Read more" button

### CODE REVIEW

File: site/gatsby-site/src/components/styles/Docs.js [hunk 1]
```diff
@@ -38,7 +38,6 @@ export const Edit = styled.div`
 `;
 
 export const StyledMainWrapper = styled.div`
-  max-width: 800px;
```
Comment (lmcnulty) [inline site/gatsby-site/src/components/styles/Docs.js:41]: This makes plain textual pages, like `/about`, [too wide to read comfortably](https://en.wikipedia.org/wiki/Line_length). I think it would be better to unset `max-width` for the blog post listing specifically.

image

Comment (lmcnulty) [reply site/gatsby-site/src/components/styles/Docs.js:41]: Otherwise, it looks good!

Comment (clari182) [reply site/gatsby-site/src/components/styles/Docs.js]: @lmcnulty fixed! Ready for another round

File: site/gatsby-site/src/templates/post.js [hunk 2]
```diff
@@ -51,7 +51,7 @@ export default function Post(props) {
           page="post"
         ></SocialShareButtons>
       </div>
-      <StyledMainWrapper>
+      <StyledMainWrapper className="max-w-full">
```
Comment (lmcnulty) [inline site/gatsby-site/src/templates/post.js:54]: I think full-width is good for the blog index, but for the posts the readability issue of long lines still applies. I would just revert the change on this line, and then it will be good to go.

Comment (clari182) [reply site/gatsby-site/src/templates/post.js]: @lmcnulty done!

**Review by lmcnulty — APPROVED** [review]
